src/v2/Apps/Fair/Components/__tests__/FairFollowedArtists.jest.tsx

```diff
@@ -0,0 +1,104 @@
+import React from "react"
+import { FairFollowedArtists } from "../FairFollowedArtists"
+import { useTracking } from "react-tracking"
+import { AnalyticsContext } from "v2/Artsy/Analytics/AnalyticsContext"
+import { mount } from "enzyme"
+import { OwnerType } from "@artsy/cohesion"
+
+jest.mock("react-tracking")
+jest.mock("v2/Components/Artwork/FillwidthItem", () => ({ onClick }) => (
+  <button onClick={onClick}>FillwidthItem</button>
+))
```

Comment:
This is how i've been doing it too, or in the case of a larger component with a lot of sub components, I mock all of the subcomponents to return null so that things are very specifically scoped to the component under test.